R/qgis-configure.R

```diff
@@ -381,18 +381,33 @@ qgis_env <- function() {
 #' @rdname qgis_run
 #' @export
 qgis_query_version <- function(quiet = FALSE) {
-  result <- qgis_run(args = character(0))
+  result <- qgis_run(args = "--version")
```

Comment:
I think it was since the moment I started investigating `qgis_process` – but that's more recent than when you created **qgisprocess**. `git` can help you find out!